src/node.cc

```diff
@@ -3067,6 +3067,7 @@ static void PrintHelp() {
          "  --v8-options          print v8 command line options\n"
 #if HAVE_OPENSSL
          "  --tls-cipher-list=val use an alternative default TLS cipher list\n"
+         "  --root-certs use specified PEM root certificate(s) file\n"
```

Comment:
not an endorsement of this PR but the text here should line up with the previous options, see how they are all formatted